pkg/consensus/consensus_test.go

```diff
@@ -259,3 +266,38 @@ func (fs *feer) NetworkFee(*transaction.Transaction) util.Fixed8 { return util.F
 func (fs *feer) IsLowPriority(util.Fixed8) bool                  { return false }
 func (fs *feer) FeePerByte(*transaction.Transaction) util.Fixed8 { return util.Fixed8(0) }
 func (fs *feer) SystemFee(*transaction.Transaction) util.Fixed8  { return util.Fixed8(0) }
+
+func addSender(t *testing.T, txs ...*transaction.Transaction) {
+	// multisig address which possess all NEO
+	scriptHash, err := util.Uint160DecodeStringBE("d60ac443bb800fb08261e75fa5925d747d485861")
+	require.NoError(t, err)
+	for _, tx := range txs {
+		tx.Sender = scriptHash
+	}
+}
+
+func signTx(t *testing.T, txs ...*transaction.Transaction) {
```

Comment:
Technically it'd be more appropriate to add signature of one of CNs, not multisig, but this code will go away soon (with Miner tx itself), so probably it's not worth fixing it now.